webui/react/src/pages/Dashboard.tsx

```diff
@@ -172,135 +177,158 @@ const Dashboard: React.FC = () => {
   }, [canceler, stopPolling]);
 
   const pageBreadCrumb: BreadCrumbRoute[] = [{ breadcrumbName: 'Home', path: paths.dashboard() }];
+
   if (projectsLoading && submissionsLoading) {
     return null;
   }
 
   return (
-    <Page
-      breadcrumb={pageBreadCrumb}
-      options={<JupyterLabButton enabled={canCreateNSC} />}
-      title="Home">
-      {projectsLoading ? (
-        <Section>
-          <Spinner center spinning />
-        </Section>
-      ) : projects.length > 0 ? (
-        // hide Projects header when empty:
-        <Section title="Recently Viewed Projects">
-          <Card.Group size="small" wrap={false}>
-            {projects.map((project) => (
-              <ProjectCard hideActionMenu key={project.id} project={project} showWorkspace />
-            ))}
-          </Card.Group>
-        </Section>
-      ) : null}
-      {/* show Submissions header even when empty: */}
-      <Section title="Your Recent Submissions">
-        {submissionsLoading ? (
-          <Spinner center spinning />
-        ) : submissions.length > 0 ? (
-          <ResponsiveTable<Submission>
-            className={css.table}
-            columns={[
-              {
-                dataIndex: 'state',
-                render: (state) => {
-                  return <ExperimentIcons state={state} />;
+    <ConditionalWrapper
+      condition={!testWithoutPage}
```

Comment:
why doesn't `Page` work while testing? i'm concerned because if there's an issue with the component, it won't be caught in our unit tests.

Reply:
thanks -- realized I did not need to separate `Page` out at all